configs/components/openssl-1.0.2.rb

```diff
@@ -95,10 +93,8 @@
   elsif platform.is_solaris?
     pkg.apply_patch 'resources/patches/openssl/add-shell-to-engines_makefile.patch'
     pkg.apply_patch 'resources/patches/openssl/openssl-1.0.0l-use-gcc-instead-of-makedepend.patch'
-  elsif platform.is_linux?
-    if platform.name =~ /debian-8-arm/
-      pkg.apply_patch 'resources/patches/openssl/openssl-1.0.0l-use-gcc-instead-of-makedepend.patch'
-    end
+  elsif platform.name =~ /debian-8-arm/
+    'linux-armv4'
```

Comment:
we no longer have a debian-8-arm platform in runtime, I suppose this can be removed